staff/henry-novoa/cv-styled/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Curriculum vitae</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="box info">
+      
+      <h2 class="personalHeader">Personal Data</h2>
+      <p class="name"> Name: Henry Novoa</p>
```

Comment:
I would put the h2 "personalHeader" inside the tag address. And each element inside "address" tag, inside a "p" tag for example.